runhouse/resources/functions/aws_lambda.py

```diff
@@ -43,13 +47,20 @@ class LambdaFunction(Function):
     DEFAULT_MEMORY_SIZE = 1024  # MB
     DEFAULT_TMP_SIZE = 3072  # MB, meaning 3G
     HOME_DIR = "/tmp/home"
+    SUPPORTED_RUNTIMES = [
```

Comment:
this is already defined in the factory file, maybe remove from there and just use this one?